tests/app/celery/test_scheduled_tasks.py

```diff
@@ -1054,7 +1054,13 @@ def test_letter_not_raise_alert_if_ack_files_match_zip_list(mocker, notify_db):
 
     letter_raise_alert_if_no_ack_file_for_zip()
 
+    yesterday = datetime.utcnow() - timedelta(days=1)
```

Comment:
Tests look good but there are five paths through the task

1. Goof path
2. Exception is thrown
3. page['Contents'] is empty
4. key.endswith(suffix.lower()) is false
5. not last_modified or obj['LastModified'] < last_modified:

Should you add more tests?

Reply:
1, 4, 5 are tested in unit tests test_get_list_of_files_by_suffix in test_s3.py
2 doesn't apply with different query. 
Added an extra test for 3 in test_s3.py